diff --git a/package.json b/package.json
index a6b1b910dcceda19e2cdf30793c015eed63537f4..a260e4c05ad5883032456c8d555f820ac5a7236c 100644
--- a/package.json
+++ b/package.json
@@ -15,12 +15,12 @@
   "module": "./dist/react-tuicss.es.js",
   "exports": {
     ".": {
-      "import": "./dist/react-tuicss.es.js",
-      "require": "./dist/react-tuicss.umd.js"
+      "types": "./dist/types/lib/main.d.ts",
+      "import": "./dist/react-tuicss.es.js"
     },
     "./style.css": {
-      "import": "./dist/style.css",
-      "require": "./dist/style.css"
+      "import": "./dist/react-tuicss.css",
+      "require": "./dist/react-tuicss.css"
     }
   },
   "scripts": {
diff --git a/dist/react-tuicss.es.js b/dist/react-tuicss.es.js
index 329d10d01f6488649397295c3905e36d214e78b6..492e47e4c19dfc968a41ab557262cf1199c9945f 100644
--- a/dist/react-tuicss.es.js
+++ b/dist/react-tuicss.es.js
@@ -1,5 +1,6 @@
 import { useCallback as e, useEffect as t, useMemo as n, useRef as r, useState as i } from "react";
 import { createPortal as a } from "react-dom";
+import { jsx as __jsx, jsxs as __jsxs, Fragment as __Fragment } from "react/jsx-runtime";
 //#region \0rolldown/runtime.js
 var o = (e, t) => () => (t || (e((t = { exports: {} }).exports, t), e = null), t.exports), s = /* @__PURE__ */ ((e) => typeof require < "u" ? require : typeof Proxy < "u" ? new Proxy(e, { get: (e, t) => (typeof require < "u" ? require : e)[t] }) : e)(function(e) {
 	if (typeof require < "u") return require.apply(this, arguments);
@@ -472,9 +473,7 @@ var o = (e, t) => () => (t || (e((t = { exports: {} }).exports, t), e = null), t
 		var Q = Z, Ve = X;
 		e.Fragment = i, e.jsx = Q, e.jsxs = Ve;
 	})();
-})), u = (/* @__PURE__ */ o(((e, t) => {
-	process.env.NODE_ENV === "production" ? t.exports = c() : t.exports = l();
-})))(), d = ({ color: e, children: t }) => /* @__PURE__ */ (0, u.jsx)("div", {
+})), u = { jsx: __jsx, jsxs: __jsxs, Fragment: __Fragment }, d = ({ color: e, children: t }) => /* @__PURE__ */ (0, u.jsx)("div", {
 	className: e,
 	children: t
 }), f = ({ textColor: e, fullWidth: t, light: r, color: i, className: a, disabled: o, ...s }) => {
